src/app/store/system-info/system-info.effects.ts

```diff
@@ -64,7 +64,7 @@ export class SystemInfoEffects {
   ));
 
   loadHaStatus = createEffect(() => this.actions$.pipe(
-    ofType(loadHaStatus, haSettingsUpdated, passiveNodeReplaced),
+    ofType(loadHaStatus, haSettingsUpdated, passiveNodeReplaced, adminUiInitialized),
```

Comment:
If we are always going to load HA status on admin ui init, we should remove `loadHaStatus` being dispatched a couple lines above.